README.md

```diff
@@ -67,6 +67,9 @@ $environment = Environment::createCommonMarkEnvironment();
 // Define your configuration:
 $config = ['html_input' => 'escape'];
 
+// To configure GitHub style newlines to create line breaks:
+$config = ['renderer' => ['soft_break' => "<br />\n"]];
```

Comment:
This line is overwriting the one above it, which may not be obvious to some users.  It would be best to provide a single `$config` array instead.